docs/specs/schema.yml

```diff
@@ -0,0 +1,33 @@
+---
+# Support schema documents in YAML format
+inputs:
+  docfx.yml:
+  docs/a.yml: |
+    #YamlMime:LandingData
+    title: my title
+    metadata:
+      key: value
+outputs:
+  docs/a.json: |
+    {
+      "content": { "title": "my title" },
```

Comment:
>content [](start = 7, length = 7)

shall we change the content to other name? confused with html content?

Reply:
html content may be an object in the future if we are to merge sdp with markdown, covered in TODO.

---
In reply to: [203633645](https://github.com/dotnet/docfx/pull/3098#discussion_r203633645) [](ancestors = 203633645)